io.github.JakubMelka.Pdf4qt.yml

```diff
@@ -0,0 +1,23 @@
+app-id: io.github.JakubMelka.Pdf4qt
+runtime: org.kde.Runtime
+runtime-version: 5.15.10
+sdk: org.freedesktop.Sdk
+command: pdf4qt
+finish-args:
+  - --socket=fallback-x11
```

Comment:
you need `share=ipc` for X11 (even fallback)

you also likely need `device=dri` if you do anything with GL.